recipe/conda_build_config.yaml

```diff
@@ -22,3 +22,9 @@ pin_run_as_build:
 docker_image:                                  # [linux64]
  - quay.io/condaforge/linux-anvil-comp7        # [linux64]
  - quay.io/condaforge/linux-anvil-cuda:11.0    # [linux64]
+
+# This needs to be removed after the pinning of libprotobuf has been adjusted.
+protobuf:
+- 3.20
```

Comment:
I would like to see it written as:
protobuf:
  - 3.20.1
... instead.  Intend is important